### PR TITLE
Prettier integration

### LINKED ISSUE #33: Prittier and linter integration.
1. Add prettier to CI.
2. Add eslint to project.

Eslint should validate indexed files using `husky` and precommit hook.

### PR DESCRIPTION
**Pull request by toplenboren**
Add prettier with default `.js style` wo semicolumns

Partially fixes #33

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Ignore everything but /apps and /packages folders
+/*
```
Comment (pahaz) [inline .prettierignore:2]: Why we need this?

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pahaz) [inline .prettierrc.json:1]: I think. It's better to put it inside `package.json`. Fewer files inside the project root look easier and less verbose for new users.

**Review by pahaz — APPROVED** [review]

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Ignore specific apps
+apps/_compability_app
```
Comment (pahaz) [inline .prettierignore:2]: @Dimitreee %)

Comment (toplenboren) [reply .prettierignore:2]: Dimitreee asked to remove this, I've just forgotten. 
Will fix

**Pull request merged**